apis/projectcontour/v1/httpproxy.go

```diff
@@ -785,9 +785,16 @@ type UpstreamValidation struct {
 type DownstreamValidation struct {
 	// Name of a Kubernetes secret that contains a CA certificate bundle.
 	// The client certificate must validate against the certificates in the bundle.
-	// +kubebuilder:validation:Required
+	// +optional
 	// +kubebuilder:validation:MinLength=1
 	CACertificate string `json:"caSecret"`
+
+	// SkipClientCertValidation disables downstream client certificate
+	// validation. Defaults to false. When external authorization is in use
+	// and is set to true, the downstream certificate is still passed to the
+	// external authorization server.
```

Comment:
I think there should be something here about how this is intended to be used with external auth, and that turning this on means that your client certs add no security value in themselves.